app/assets/javascripts/tour.js

```diff
@@ -0,0 +1,87 @@
+const Tether = require('tether');
```

Comment:
We're not using using Babel on the project yet (@eddiecode  is including it in a PR he's working) so we should avoid const and lambda until his PR is merged.